src/common/Tile/styled.js

```diff
@@ -77,28 +77,28 @@ export const Tile = styled.div`
 export const Image = styled.img`
```

Comment:
If you changed height to auto in main `Image` style, you can delete other ` height: auto;` 😄 just a small note not to duplicate the same code 😃